### ISSUE TITLE
How to migrate both vm and storage using govc commands.

### ISSUE THREAD

**Issue by HaruChebrolu** [vmware/govmomi#3415]
I want to know if there are govc commands are available for vMotion covering below scenarios.
<img width="826" alt="Screenshot 2024-04-12 at 1 58 43 PM" src="https://github.com/vmware/govmomi/assets/97601751/d5be6e40-76da-4bd4-a5be-1da41dd16f0f">

**Comment by dougm** [comment]
Yes, see [vm.migrate](https://github.com/vmware/govmomi/blob/main/govc/USAGE.md#vmmigrate)

**Comment by HaruChebrolu** [comment]
Okay @dougm, will check!
Thank you!

**Comment by HaruChebrolu** [comment]
But, I don't see an option to mention destination network when we migrate vm using [vm.migrate](https://github.com/vmware/govmomi/blob/main/govc/USAGE.md#vmmigrate)
<img width="1126" alt="Screenshot 2024-04-17 at 9 21 10 PM" src="https://github.com/vmware/govmomi/assets/97601751/46c20e3a-56a9-463f-b997-991ee142ba3a">

**Comment by HaruChebrolu** [comment]
@dougm Do we have any update on this.
